hrstaffdir/tests/phpunit/WebTest/HRStaffDir/HRStaffDirProfile.php

```diff
@@ -24,7 +24,6 @@
  +--------------------------------------------------------------------+
 */
 
-require_once 'CiviTest/CiviSeleniumTestCase.php';
```

Comment:
I think you didn't try to run these tests, right? Besides removing the require_once, you also have to add the bootstrap.php and the phpunit.xml.dist files to the extensions.